master/internal/job/job.go

```diff
@@ -102,8 +102,20 @@ func (j *Manager) GetJobs(
 		return nil, err
 	}
 
+	jobCount := len(jobQ)
+
+	// Try to fetch Non-DAI jobs if supported by the Resource Manager.
+	// If the Resource Manager returned Non-DAI jobs, increase the job count.
+	// Otherwise, we do not show any Non-DAI jobs.
+	nonDaiJobs, err := j.rm.GetNonDaiJobs(j.system, sproto.GetNonDaiJobs{
+		ResourcePool: resourcePool,
+	})
+	if err == nil {
+		jobCount += len(nonDaiJobs)
+	}
+
 	// Merge the results.
-	jobsInRM := make([]*jobv1.Job, 0, len(jobQ))
+	jobsInRM := make([]*jobv1.Job, 0, jobCount)
```

Comment:
i dont see why this calc needs to be so spread out. having stuff closer to where it is needed is usually easier to read.
```suggestion
        jobCount := len(jobQ) + len(nonDaiJobs)
	jobsInRM := make([]*jobv1.Job, 0, jobCount)
```

Reply:
Updated as suggested.